Filenames refactor - part VII

komret opened the pull request:
<!--- Provide a general summary of your changes in the Title above -->

## Description

<!--- Describe your changes in detail -->
- Filenames and component names are contextualised, e.g. `Stack` renamed to `SkeletonStack`.
- Unused components deleted (`DeviceIcon`, `HelperTooltip`).
- Folder structure reorganised so that it resembles component structure.
- Filepaths shortened by removing nested components directories.
- Default exports replaced with named exports.
- Index files renamed to bear the component's name.
- Types moved to related component files (`TransactionsGraph).
- Props destructured.

`src/components/suite` and `src/components/wallet` refactoring is now finished, I'll continue to other directories later.

## Related Issue

[Part VI](https://github.com/trezor/trezor-suite/pull/9640)

dahaca commented on packages/suite/src/components/suite/Translation.tsx:
```diff
@@ -23,14 +25,14 @@ export const isMsgType = (
     typeof props === 'object' && props !== null && (props as MsgType).id !== undefined;
 
-const BaseTranslation = (props: MsgType) => {
+export const Translation = (props: MsgType) => {
```
I believe `props.isReactNative` is also unused and never will be.

dahaca reviewed (approved):
Since you've been doing this for a while, could you add a paragraph about the proj structure to the style guide sometime soon? I bet you came up with some nice patterns for Suite and been following them yourself throughout all those PRs :)

komret commented:
The guide is on my mind, I will update it when it is clear what to do with the views directory.

I removed the `isReactNative` prop and rebased on develop to resolve conflicts.

komret commented:
/rebase
